docs/tutorial/force_calibration.rst

```diff
@@ -1,380 +1,375 @@
 
-    \kappa = 2 \pi \gamma_0 f_c \quad \mathrm{[N/m]}
+.. image:: figures/force_calibration/power_spectrum.png
 
-Here :math:`\kappa` then represents the final trap stiffness.
+The power spectrum is smoothed by downsampling adjacent power spectral values.
+Downsampling the spectrum is required to fulfill some of the assumptions in the fitting procedure, but it comes at the cost of spectral resolution.
+One must be careful that the shape of the power spectrum is still sufficiently preserved.
+If the corner frequency is very low then downsampling too much can lead to biases in the calibration parameters.
+In such cases, it is better to either measure a longer interval to increase the spectral resolution or reduce the number of points (`num_points_per_block`) used for blocking.
 
-We can calibrate the position by considering the diffusion of the bead:
+When calculating the power spectrum, the amount of downsampling (`num_points_per_block`) and the range over which to compute the spectrum (`fit_range`) can be controlled.
+One can also exclude specific frequency ranges from the spectrum (`excluded_ranges`) which can be useful if there are noise peaks in the spectrum.
+Let's see which ranges were excluded in our Bluelake calibration::
 
-.. math::
+    force_slice.calibration[0]
 
-    D_\mathrm{physical} = \frac{k_B T}{\gamma_0} \quad \mathrm{[m^2/s]}
+.. image:: figures/force_calibration/bl_dictionary.png
 
-Here :math:`k_B` is the Boltzmann constant and :math:`T` is the local temperature in Kelvin. Comparing this to its measured counterpart in volts squared per second provides us with the desired calibration factor:
+Here, they are listed as `Exclusion range 0 (min.) (Hz)`, `Exclusion range 0 (max.) (Hz)` etc.
+To reproduce the result obtained with Bluelake, these should be excluded from the power spectrum::
 
-.. math::
+    power_spectrum = lk.calculate_power_spectrum(volts.data, sample_rate=volts.sample_rate, fit_range=(1e2, 23e3), num_points_per_block=2000, excluded_ranges=([19348, 19668], [24308, 24548]))
+    power_spectrum.plot(marker=".")
 
-    R_d = \sqrt{\frac{D_\mathrm{physical}}{D_\mathrm{measured}}} \quad \mathrm{[m/V]}
+.. image:: figures/force_calibration/power_spectrum_excluded_ranges.png
 
-Both of these quantities depend on the parameter :math:`\gamma_0`, which corresponds to the drag coefficient of a sphere and is given by:
+Note that exclusion ranges are excluded _prior_ to downsampling.
+Considering that a noise peak may be very narrow, it is beneficial to lower the number of points per block temporarily to find the exact exclusion range.
+After determination of this exclusion range, the number of points per block can be increased again.
 
-.. math::
+Passive calibration
+-------------------
 
-    \gamma_0 = 3 \pi \eta d \quad \mathrm{[kg/s]}
+In literature, passive calibration is often referred to as thermal calibration.
+It involves fitting a physical model to the power spectrum obtained in the previous step.
```

Comment:
Yeah, I kept it generic on purpose, since I wanted to keep the terminology to a minimum. It's also not just a Lorentzian. It's a Lorentzian times a filtering function in the simplest case. I'd rather rephrase it a bit in the fast sensor section to not mention the Lorentzian, but refer to it as the simple model. What do you think?

Reply:
maybe could work. You use "Lorentzian" a lot in the theory section too. So either replacing the term everywhere or defining it somewhere I'd be ok with

Reply:
I've explicitly adding a sentence defining it in the theory section. I think using terminology in the section where people are interested in theory is more acceptable, no?